### PR TITLE
feat: Implementation Considerations for Offline Flows using Trust Chain JWT header parameter

### PR DESCRIPTION
**Pull request by peppelinux**
This PR aims to resolve the undone section still with TBD placeholder.

### CODE REVIEW

File: openid-federation-wallet-1_0.md [hunk 1]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers is used to ensure that the entities involved in the transaction have a common Trust Anchor. which facilitates trust without the need for real-time verification using the Federation API endpoints.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:567]:
```suggestion
The static Trust Chain parameter within the JWT headers is used to ensure that the entities involved in the transaction have a common Trust Anchor. This enables trust without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```

File: openid-federation-wallet-1_0.md [hunk 2]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers is used to ensure that the entities involved in the transaction have a common Trust Anchor. This enables trust without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:567]:
```suggestion
The static Trust Chain parameter within the JWT headers, as defined in [@!OpenID.Federation] Section 4.3, is used to hint to the entity involved in a transaction to have a common Trust Anchor. This facilitate the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```

File: openid-federation-wallet-1_0.md [hunk 3]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers, as defined in [@!OpenID.Federation] Section 4.3, is used to hint to the entity involved in a transaction to have a common Trust Anchor. This facilitate the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
+
+Entity that issues a signed data object, including the `trust_chain` parameter, might be:
+
+- Wallet Providers in signed Wallet Attestations. The Wallet Instance obtains one or more Wallet Attestations from its Wallet Provider, each of them including a Trust Chain related to each Trust Anchor the Wallet Providers trusts;
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:571]:
```suggestion
- Wallet Providers in signed Wallet Attestations. The Wallet Instance obtains one or more Wallet Attestations from its Wallet Provider, each of them including a Trust Chain related to each Trust Anchor the Wallet Provider trusts;
```

File: openid-federation-wallet-1_0.md [hunk 4]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers, as defined in [@!OpenID.Federation] Section 4.3, is used to hint to the entity involved in a transaction to have a common Trust Anchor. This facilitate the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
+
+Entity that issues a signed data object, including the `trust_chain` parameter, might be:
+
+- Wallet Providers in signed Wallet Attestations. The Wallet Instance obtains one or more Wallet Attestations from its Wallet Provider, each of them including a Trust Chain related to each Trust Anchor the Wallet Provider trusts;
+- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request which includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider;
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:572]:
```suggestion
- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request which includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the `wallet_metadata` parameter provided by the Wallet in the Request URI POST;
```

File: openid-federation-wallet-1_0.md [hunk 5]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
...
+- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request which includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the `wallet_metadata` parameter provided by the Wallet in the Request URI POST;
+- Credential Issuers in signed Digital Credential. The Wallet Instance obtains a Digital Credential from its Credential Issuer, which includes the Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the Wallet Attestation used during the Issuance.
+
+The Entity that receives the data object including the JWT `trust_chain`, such as the Wallet Instance obtaining a signed request object, verifies the Trust Chain using the Trust Anchor's public keys and applies any metadata policies, without needing to have a working internet connection for reaching the Federation API.
+
+Using short-lived Trust Chains ensures compatibility with required revocation administrative protocols. For example, if a revocation must be propagated in less than 24 hours, the Trust Chain should not be valid for more than that period.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:577]:
```suggestion
Using short-lived Trust Chains ensures compatibility with required revocation administrative protocols, such as those defined in a legal framework. For example, if a revocation must be propagated in less than 24 hours, the Trust Chain should not be valid for more than that period.
```

File: openid-federation-wallet-1_0.md [hunk 6]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers, as defined in [@!OpenID.Federation] Section 4.3, is used to hint to the entity involved in a transaction to have a common Trust Anchor. This facilitate the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:567]:
```suggestion
The static Trust Chain parameter within the JWT headers, as defined in Section 4.3 of [@!OpenID.Federation], is used as a hint to the Entity involved in a transaction to have a common Trust Anchor. This facilitates the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```

File: openid-federation-wallet-1_0.md [hunk 7]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
...
+
+Entity that issues a signed data object, including the `trust_chain` parameter, might be:
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:569]:
```suggestion
The Entity that issues a signed data object, including the `trust_chain` parameter, might be:
```

File: openid-federation-wallet-1_0.md [hunk 8]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
...
+
+- Wallet Providers in signed Wallet Attestations. The Wallet Instance obtains one or more Wallet Attestations from its Wallet Provider, each of them including a Trust Chain related to each Trust Anchor the Wallet Provider trusts;
+- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request which includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the `wallet_metadata` parameter provided by the Wallet in the Request URI POST;
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:572]:
```suggestion
- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request that includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the `wallet_metadata` parameter provided by the Wallet in the Request URI POST;
```

File: openid-federation-wallet-1_0.md [hunk 9]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
...
+- Credential Issuers in signed Digital Credential. The Wallet Instance obtains a Digital Credential from its Credential Issuer, which includes the Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the Wallet Attestation used during the Issuance.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:573]:
```suggestion
- A Credential Issuer in a signed Digital Credential. The Wallet Instance obtains a Digital Credential from its Credential Issuer, which includes the Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the Wallet Attestation used during the Issuance.
```

**Review by selfissued — APPROVED** [review]
Approved with suggested changes.

File: openid-federation-wallet-1_0.md [hunk 10]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
 
 # Implementation Considerations for Offline Flows
 
-TBD: usage of static trust chain having at least a Trust Anchor in common with the requestor
+The static Trust Chain parameter within the JWT headers, as defined in Section 4.3 of [@!OpenID.Federation], is used as a hint to the Entity involved in a transaction to have a common Trust Anchor. This facilitates the trust evaluation without the need for real-time Federation Entity Discovery using the Federation API endpoints.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:567]:
```suggestion
The static Trust Chain parameter within the JWT headers, as defined in Section 4.3 of [@!OpenID.Federation], is used as a hint to the Entity involved in a transaction with a common Trust Anchor. This facilitates trust evaluation without the need for real-time Federation Entity Discovery using Federation API endpoints.
```

File: openid-federation-wallet-1_0.md [hunk 11]
```diff
@@ -566,7 +564,18 @@ sequenceDiagram
...
+
+- Wallet Providers in signed Wallet Attestations. The Wallet Instance obtains one or more Wallet Attestations from its Wallet Provider, each of them including a Trust Chain related to each Trust Anchor the Wallet Provider trusts;
+- Credential Verifiers in signed request objects. The Wallet Instance obtains a presentation request that includes a Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the `wallet_metadata` parameter provided by the Wallet in the Request URI POST;
+- A Credential Issuer in a signed Digital Credential. The Wallet Instance obtains a Digital Credential from its Credential Issuer, which includes the Trust Chain using a Trust Anchor that the Credential Verifier has in common with the Wallet Provider, according to the Wallet Attestation used during the Issuance.
+
+The Entity that receives the data object including the JWT `trust_chain`, such as the Wallet Instance obtaining a signed request object, verifies the Trust Chain using the Trust Anchor's public keys and applies any metadata policies, without needing to have a working internet connection for reaching the Federation API.
```
Comment (selfissued) [inline openid-federation-wallet-1_0.md:575]:
```suggestion
The Entity that receives the data object including the JWT `trust_chain`, such as the Wallet Instance obtaining a signed request object, verifies the Trust Chain using the Trust Anchor's public keys and applies any metadata policies, without needing to have a working network connection for reaching the Federation API.
```

**Pull request merged**